lib/linters/url_quotes.js

```diff
@@ -34,6 +34,11 @@ module.exports = {
             const uri = child.first.next();
             const value = uri.value.trim();
 
+            // Ignore variables
+            if (uri.value.substring(0, 1) === '@') {
```

Comment:
Could you switch this to just `value.substring(...)`? `url( @bar )` will still error.